src/components/TablePagination/TablePagination.stories.mdx

```diff
@@ -0,0 +1,214 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+
+import TablePagination from "./TablePagination";
+import MainTable from "../MainTable";
+
+<Meta title="TablePagination" component={TablePagination} />
+
+export const Template = (args) => <TablePagination {...args} />;
+
+### TablePagination
+
+This is an HOC [React](https://reactjs.org/) component for applying pagination to input data for direct child components.
+This component is un-opinionated about the structure of the input data and can be used with any child component that displays 
+a list of data.
```

Comment:
Probably should mention this is meant for tables and works nicely with `MainTable`?

Reply:
Sure, that makes sense :+1:

Reply:
This is an HOC [React](https://reactjs.org/) component for applying pagination to input data for direct child components.
This component is un-opinionated about the structure of the input data and can be used with any child component that displays a list of data. However, the styling and behaviour of this component were designed to work nicely with the ```MainTable``` component.

Reply:
sounds good to me.